Formula/pygobject3.rb

```diff
@@ -11,14 +11,15 @@ class Pygobject3 < Formula
     sha256 "4c637d654502521ef19d7679a159ec353115df01d1afd3934ea43799d3b79b39" => :el_capitan
   end
 
-  option "without-python", "Build without python2 support"
+  option "without-python@2", "Build without python2 support"
```

Comment:
This is causing `reinstall` and `upgrade` failures for me as-is, having built my `pygobject3` formula with `python3` support.

```
pygobject3: stable 3.26.1 (bottled)
GNOME Python bindings (based on GObject Introspection)
https://live.gnome.org/PyGObject
/usr/local/Cellar/pygobject3/3.26.1 (159 files, 3.5MB) *
  Built from source on 2017-10-27 at 20:04:32 with: --with-python3
```

```
~> brew upgrade -s -v pygobject3
==> Upgrading 1 outdated package, with result:
pygobject3 3.26.1_1
==> Upgrading pygobject3
rm /usr/local/include/pygobject-3.0
rm /usr/local/lib/pkgconfig/pygobject-3.0.pc
rm /usr/local/lib/python2.7/site-packages/gi/__init__.py
rm /usr/local/lib/python2.7/site-packages/gi/_constants.py
rm /usr/local/lib/python2.7/site-packages/gi/_error.py
rm /usr/local/lib/python2.7/site-packages/gi/_gi.so
rm /usr/local/lib/python2.7/site-packages/gi/_gi_cairo.so
rm /usr/local/lib/python2.7/site-packages/gi/_option.py
rm /usr/local/lib/python2.7/site-packages/gi/_propertyhelper.py
rm /usr/local/lib/python2.7/site-packages/gi/_signalhelper.py
rm /usr/local/lib/python2.7/site-packages/gi/docstring.py
rm /usr/local/lib/python2.7/site-packages/gi/importer.py
rm /usr/local/lib/python2.7/site-packages/gi/module.py
rm /usr/local/lib/python2.7/site-packages/gi/overrides/GIMarshallingTests.py
rm /usr/local/lib/python2.7/site-packages/gi/overrides/GLib.py
rm /usr/local/lib/python2.7/site-packages/gi/overrides/GObject.py
rm /usr/local/lib/python2.7/site-packages/gi/overrides/Gdk.py
rm /usr/local/lib/python2.7/site-packages/gi/overrides/Gio.py
rm /usr/local/lib/python2.7/site-packages/gi/overrides/Gtk.py
rm /usr/local/lib/python2.7/site-packages/gi/overrides/Pango.py
rm /usr/local/lib/python2.7/site-packages/gi/overrides/__init__.py
rm /usr/local/lib/python2.7/site-packages/gi/overrides/keysyms.py
rm /usr/local/lib/python2.7/site-packages/gi/pygtkcompat.py
rm /usr/local/lib/python2.7/site-packages/gi/repository/__init__.py
rm /usr/local/lib/python2.7/site-packages/gi/types.py
rm /usr/local/lib/python2.7/site-packages/pygobject-3.26.1-py2.7.egg-info
rm /usr/local/lib/python2.7/site-packages/pygtkcompat/__init__.py
rm /usr/local/lib/python2.7/site-packages/pygtkcompat/generictreemodel.py
rm /usr/local/lib/python2.7/site-packages/pygtkcompat/pygtkcompat.py
rm /usr/local/lib/python3.6/site-packages/gi/__init__.py
rm /usr/local/lib/python3.6/site-packages/gi/_constants.py
rm /usr/local/lib/python3.6/site-packages/gi/_error.py
rm /usr/local/lib/python3.6/site-packages/gi/_gi.cpython-36m-darwin.so
rm /usr/local/lib/python3.6/site-packages/gi/_gi_cairo.cpython-36m-darwin.so
rm /usr/local/lib/python3.6/site-packages/gi/_option.py
rm /usr/local/lib/python3.6/site-packages/gi/_propertyhelper.py
rm /usr/local/lib/python3.6/site-packages/gi/_signalhelper.py
rm /usr/local/lib/python3.6/site-packages/gi/docstring.py
rm /usr/local/lib/python3.6/site-packages/gi/importer.py
rm /usr/local/lib/python3.6/site-packages/gi/module.py
rm /usr/local/lib/python3.6/site-packages/gi/overrides/GIMarshallingTests.py
rm /usr/local/lib/python3.6/site-packages/gi/overrides/GLib.py
rm /usr/local/lib/python3.6/site-packages/gi/overrides/GObject.py
rm /usr/local/lib/python3.6/site-packages/gi/overrides/Gdk.py
rm /usr/local/lib/python3.6/site-packages/gi/overrides/Gio.py
rm /usr/local/lib/python3.6/site-packages/gi/overrides/Gtk.py
rm /usr/local/lib/python3.6/site-packages/gi/overrides/Pango.py
rm /usr/local/lib/python3.6/site-packages/gi/overrides/__init__.py
rm /usr/local/lib/python3.6/site-packages/gi/overrides/keysyms.py
rm /usr/local/lib/python3.6/site-packages/gi/pygtkcompat.py
rm /usr/local/lib/python3.6/site-packages/gi/repository/__init__.py
rm /usr/local/lib/python3.6/site-packages/gi/types.py
rm /usr/local/lib/python3.6/site-packages/pygobject-3.26.1-py3.6.egg-info
rm /usr/local/lib/python3.6/site-packages/pygtkcompat/__init__.py
rm /usr/local/lib/python3.6/site-packages/pygtkcompat/generictreemodel.py
rm /usr/local/lib/python3.6/site-packages/pygtkcompat/pygtkcompat.py
rmdir /usr/local/lib/python3.6/site-packages/pygtkcompat/__pycache__
rmdir /usr/local/lib/python3.6/site-packages/pygtkcompat
rmdir /usr/local/lib/python3.6/site-packages/gi/repository/__pycache__
rmdir /usr/local/lib/python3.6/site-packages/gi/repository
rmdir /usr/local/lib/python3.6/site-packages/gi/overrides/__pycache__
rmdir /usr/local/lib/python3.6/site-packages/gi/overrides
rmdir /usr/local/lib/python2.7/site-packages/pygtkcompat
/usr/bin/sandbox-exec -f /tmp/homebrew20180301-42809-1xvjxwm.sb nice /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby -W0 -I /usr/local/Homebrew/Library/Homebrew -- /usr/local/Homebrew/Library/Homebrew/build.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/pygobject3.rb --verbose
==> Downloading https://download.gnome.org/sources/pygobject/3.26/pygobject-3.26.1.tar.xz
Already downloaded: /usr/local/var/homebrewcache/pygobject3-3.26.1.tar.xz
==> Verifying pygobject3-3.26.1.tar.xz checksum
tar xJf /usr/local/var/homebrewcache/pygobject3-3.26.1.tar.xz
ln -s ../Cellar/pygobject3/3.26.1/include/pygobject-3.0 pygobject-3.0
ln -s ../../Cellar/pygobject3/3.26.1/lib/pkgconfig/pygobject-3.0.pc pygobject-3.0.pc
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/__init__.py __init__.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/_constants.py _constants.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/_error.py _error.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/_gi.so _gi.so
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/_gi_cairo.so _gi_cairo.so
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/_option.py _option.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/_propertyhelper.py _propertyhelper.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/_signalhelper.py _signalhelper.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/docstring.py docstring.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/importer.py importer.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/module.py module.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/overrides/GIMarshallingTests.py GIMarshallingTests.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/overrides/GLib.py GLib.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/overrides/GObject.py GObject.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/overrides/Gdk.py Gdk.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/overrides/Gio.py Gio.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/overrides/Gtk.py Gtk.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/overrides/Pango.py Pango.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/overrides/__init__.py __init__.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/overrides/keysyms.py keysyms.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/pygtkcompat.py pygtkcompat.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/repository/__init__.py __init__.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/gi/types.py types.py
ln -s ../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/pygobject-3.26.1-py2.7.egg-info pygobject-3.26.1-py2.7.egg-info
mkdir -p /usr/local/lib/python2.7/site-packages/pygtkcompat
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/pygtkcompat/__init__.py __init__.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/pygtkcompat/generictreemodel.py generictreemodel.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python2.7/site-packages/pygtkcompat/pygtkcompat.py pygtkcompat.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/__init__.py __init__.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/_constants.py _constants.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/_error.py _error.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/_gi.cpython-36m-darwin.so _gi.cpython-36m-darwin.so
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/_gi_cairo.cpython-36m-darwin.so _gi_cairo.cpython-36m-darwin.so
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/_option.py _option.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/_propertyhelper.py _propertyhelper.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/_signalhelper.py _signalhelper.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/docstring.py docstring.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/importer.py importer.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/module.py module.py
mkdir -p /usr/local/lib/python3.6/site-packages/gi/overrides
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/overrides/GIMarshallingTests.py GIMarshallingTests.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/overrides/GLib.py GLib.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/overrides/GObject.py GObject.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/overrides/Gdk.py Gdk.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/overrides/Gio.py Gio.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/overrides/Gtk.py Gtk.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/overrides/Pango.py Pango.py
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/overrides/__init__.py __init__.py
mkdir -p /usr/local/lib/python3.6/site-packages/gi/overrides/__pycache__
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/overrides/keysyms.py keysyms.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/pygtkcompat.py pygtkcompat.py
mkdir -p /usr/local/lib/python3.6/site-packages/gi/repository
ln -s ../../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/repository/__init__.py __init__.py
mkdir -p /usr/local/lib/python3.6/site-packages/gi/repository/__pycache__
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/gi/types.py types.py
ln -s ../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/pygobject-3.26.1-py3.6.egg-info pygobject-3.26.1-py3.6.egg-info
mkdir -p /usr/local/lib/python3.6/site-packages/pygtkcompat
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/pygtkcompat/__init__.py __init__.py
mkdir -p /usr/local/lib/python3.6/site-packages/pygtkcompat/__pycache__
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/pygtkcompat/generictreemodel.py generictreemodel.py
ln -s ../../../../Cellar/pygobject3/3.26.1/lib/python3.6/site-packages/pygtkcompat/pygtkcompat.py pygtkcompat.py
Error: Empty installation
```